xc/device.mli

```diff
@@ -235,7 +235,11 @@ sig
 	val stop : xs:Xenstore.Xs.xsh -> qemu_domid:int -> Xenctrl.domid -> unit
 
 	val maybe_write_pv_feature_flags : xs:Xenstore.Xs.xsh -> Xenctrl.domid -> unit
+	
+	val init_qmp_event : unit -> unit
```

Comment:
no need to export init_qmp_event, it doesn't need to be a function.

Reply:
I think we need, see my answer above, thanks.